docker/admin/rebuild-ilios.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I'm very worried about this file. Doing a composer install feels like a code smell to me. The bastion container should be destroyed and re-created. Preferable whenever it is needed. Which will result in a fresh composer install anyway. There is a purpose built container for running migrations as well. Doing any of this work in a long lived place is going to result in issues where the code here is out of sync with the latest release.